packages/contracts/as/NFT/assembly/index.ts

```diff
@@ -322,6 +327,93 @@ export function nft_revoke_all(token_id: string): void {
     }
 }
 
+/* Storage management */
+
+function internal_storage_balance_of(
+    account_id: string
+): StorageBalance | null {
+    if (accounts.get(account_id)) {
+        return <StorageBalance>{
+            total: storage_balance_bounds().min,
+            available: '0',
```

Comment:
I understand this is 0 for Fungible Tokens, cause supposedly user will have already paid his storage, if accounts exist, so he will not have an available balance. I don't think it will be the same for NFTs which is closer to the example you have in the Nomicon of a [Social Media Contract](https://nomicon.io/Standards/StorageManagement.html#example-2-social-media-contract).